engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/GetMultiValuesBuiltin.java

```diff
@@ -0,0 +1,53 @@
+package org.enso.interpreter.node.expression.builtin.meta;
+
+import com.oracle.truffle.api.interop.UnsupportedMessageException;
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.dsl.AcceptsWarning;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.runtime.EnsoContext;
+import org.enso.interpreter.runtime.data.EnsoMultiValue;
+import org.enso.interpreter.runtime.data.EnsoObject;
+import org.enso.interpreter.runtime.data.Type;
+import org.enso.interpreter.runtime.data.vector.ArrayLikeHelpers;
+import org.enso.interpreter.runtime.library.dispatch.TypeOfNode;
+import org.enso.interpreter.runtime.warning.WarningsLibrary;
+
+@BuiltinMethod(
+    type = "Meta",
+    name = "get_multi_values",
```

Comment:
is this name valid? Shouldn't this be more like `get_multi_types`?

Reply:
You are probably right, I will rename it.